test/server/cards/01_SOR/events/Waylay.spec.js

```diff
@@ -0,0 +1,53 @@
+describe('Waylay', function() {
+    integration(function() {
+        describe('Waylay\'s ability', function() {
+            beforeEach(function () {
+                this.setupTest({
+                    phase: 'action',
+                    player1: {
+                        hand: ['waylay', 'waylay'],
+                        groundArena: ['pyke-sentinel'],
+                    },
+                    player2: {
+                        hand: ['entrenched'],
+                        groundArena: ['wampa', 'superlaser-technician'],
+                        spaceArena: [{ card: 'imperial-interceptor', upgrades: ['academy-training'] }],
+                        leader: { card: 'grand-moff-tarkin#oversector-governor', deployed: true }
+                    }
+                });
+            });
+
+            it('can return a friendly or enemy unit to its owner\'s hand', function () {
+                this.player1.clickCard('waylay');
+                expect(this.player1).toBeAbleToSelectExactly([this.wampa, this.pykeSentinel, this.imperialInterceptor, this.superlaserTechnician]);
+
+                this.player1.clickCard(this.superlaserTechnician);
+                expect(this.superlaserTechnician).toBeInLocation('hand', this.player2);
+            });
+
+            it('should allow the player to select a friendly unit to return to hand and should remove damage and be playable', function () {
+                this.pykeSentinel.damage = 2;
+                this.pykeSentinel.exhausted = false; // Making sure it doesn't retain state of unexhausted
```

Comment:
Does it work without this? Since other people tend to base their tests on existing ones, I want to avoid adding unnecessary code so that people don't think it's required for things to work right

Reply:
I can add do not copy. To the comment. But I just wanted to be sure testing this mechanic, if you think it's unnecessary I'm happy to remove it but I'm more inclined to add a THIS IS NOT NECESSARY FOR OTHER TESTS to the comment

Reply:
Hmm I might be misunderstanding. I'm trying to say that the unit should start the test ready and this line is redundant. If that's not true there's a problem.